torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9366,7 +9366,10 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
     OpInfo('resize_',
            op=lambda x, shape: x.clone().resize_(shape),
            method_variant=None,
-           inplace_variant=None,
+           inplace_variant=torch.Tensor.resize_,
+           # the test fails because resize_ doesn't work with imag views as expected by the test
```

Comment:
this is expected behavior (see Alban's comment in #65945)

Reply:
Yes, that's why the issue is linked here for anyone in the future to know why test_neg_view is skipped.